### ISSUE TITLE
Support rich text copy into clipboard

### ISSUE THREAD

**Issue by SnosMe** [PathOfBuildingCommunity/PathOfBuilding#4044]
### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Is your feature request related to a problem?

PoB copies to clipboard text in source format.

### Describe the solution you'd like

Copy text to clipboard in multiple formats 
https://docs.microsoft.com/en-us/windows/win32/dataxchg/clipboard-formats#multiple-clipboard-formats
Do the same thing browsers do, after doing a copy you can paste it in MS Word as plain text or keep formatting.
![image](https://user-images.githubusercontent.com/4292308/152318958-1022cf28-7c6c-40b5-ba61-906f24902a06.png)

### Describe alternatives you've considered

Create distinct copy buttons, keep current behavior unchanged.

**Comment by pHiney** [comment]
As there are no font options, the only advantage to this is the colours. Is this what you expect ?
X

**Comment by SnosMe** [comment]
Yes, I'm not requesting for more formatting options or pasting formatted text from other programs.
Only copying text FROM PoB in away that not a PoB specific.

**Comment by pHiney** [comment]
I've done some research. We will need to update SimpleGraphic.dll to support the RTF clipboard format. It is hard coded to Text format at the moment.

**Comment by pHiney** [comment]
Until simplegraphic is updated to support RTF, will saving the Notes as a RTF file do ? One extra step I understand.
